src/ssl.c

```diff
@@ -46222,7 +46233,7 @@ unsigned long wolfSSL_ERR_peek_error_line_data(const char **file, int *line,
                 ret = -ret;
             }
 
-            if (ret == ASN_NO_PEM_HEADER)
+            if (ret == -ASN_NO_PEM_HEADER)
```

Comment:
Is this always correct, a - error?  Shouldn't we convert the error before checking?

Reply:
The if case above this one takes the absolute values and so does `WOLFSSL_ERROR_LINE` in line 407: `error = error - (2 * error); /* get absolute value */`.
Checking this is important because in `wolfSSL_PEM_read_bio_X509` we push `ASN_NO_PEM_HEADER` as an error to mean that the `WOLFSSL_BIO` is empty. This error is used in `src/openvpn/ssl_openssl.c` in `tls_ctx_add_extra_certs`.